src/plugins/presentation_util/public/components/controls/control_types/options_list/options_list_embeddable.tsx

```diff
@@ -58,22 +61,19 @@ export const OPTIONS_LIST_CONTROL = 'optionsListControl';
 export interface OptionsListEmbeddableInput extends InputControlInput {
   field: string;
   indexPattern: string;
-  multiSelect: boolean;
+  singleSelect?: boolean;
```

Comment:
just curious: what's the reason for this flip?

Reply:
This is flipped because `multiselect` defaults to true. Making an optional `singleSelect` entry in the input is the easiest way to handle that because undefined is falsy. If you want `multiselect` you can just pass nothing! 

It's the perfect time right for renaming input keys, because once this is released we'd have to write migrations for changes like this.